website/src/content/docs/docs/extending-typespec/emitter-framework.md

```diff
@@ -69,6 +69,13 @@ export async function $onEmit(context: EmitContext) {
 
 If you run this emitter, you will see a `src` directory and a `README.md` file in your `tsp-output` directory.
 
+## Core Emitter Framework Components
+
+The emitter framework provides two components to allow easy access to typekits from within components:
+
+- `<TspContext.Provider value={Program}>` - accepts a TypeSpec `Program`, and makes it and ready to use typekits available in any child components via the `useTsp()` hook.
+- `<Output program={Program}>` - wrapper around the core alloy `Output` component that additionally takes a TypeSpec `Program` to allow use of the `useTsp()` hook in any child component.
```

Comment:
show this one as first as its probably what you should use?

Reply:
Good call!